core/commands/keystore.go

```diff
@@ -112,12 +115,20 @@ var keyGenCmd = &cmds.Command{
 	Type: KeyOutput{},
 }
 
+func formatID(id peer.ID, useCID bool) string {
+	if useCID {
+		return b36.EncodeToStringLc(peer.ToCid(id).Bytes())
```

Comment:
Is there a reason to use this more explicit form rather than `peer.ToCid(id).StringOfBase(mbase.Base36)`?

Reply:
I missed that method. Fixed.